mmpose/core/post_processing/post_transforms.py

```diff
@@ -75,6 +75,32 @@ def fliplr_regression(regression, flip_pairs):
     return regression_flipped
 
 
+def fliplr_regression_3d(regression, flip_pairs):
+    """Flip human joints horizontally.
+
+    Note:
+        batch_size: N
+        num_keypoint: K
+    Args:
+        regression (np.ndarray([N, K, 3])): Coordinates of keypoints.
+        flip_pairs (list[tuple()]): Pairs of keypoints which are mirrored
+            (for example, left ear -- right ear).
+
+    Returns:
+        tuple: Flipped human joints.
+
+        - regression_flipped (np.ndarray([N, K, 3])): Flipped joints.
+    """
+    regression_flipped = regression.copy()
+    # Swap left-right parts
+    for left, right in flip_pairs:
+        regression_flipped[:, left, :] = regression[:, right, :]
+        regression_flipped[:, right, :] = regression[:, left, :]
+    # Flip horizontally
+    regression_flipped[:, :, 0] = -regression_flipped[:, :, 0]
```

Comment:
Double check this transformation to make sure that there are no issues concerning the absolute coordinates of pelvis

Reply:
Confirmed in person that this is intended.